### ISSUE TITLE
Settings page updates

### ISSUE THREAD

**Issue by mikeshultz** [OriginProtocol/origin#2333]
@micahalcorn has requested changes to the settings page that includes:

- Style changes
- Removal of browser notification disable button
- Removal of performance mode toggle
- A "Developer Mode" toggle to hide service endpoints
- Addition of a way to disable messaging 
- Addition of "Decentralized Mode"

## Done

- Design changes (mostly, see below)

## TODO/Needs

- The down-caret icon svg still needs to be provided
- Decentralied Mode needs to be implemented (ref #2320)
- Disable Messaging still needs to be implemented

**Comment by micahalcorn** [comment]
Just a note to say that #2320 should not hold up the rest of this ⏩

**Comment by mikeshultz** [comment]
I think you mentioned that, but yeah, this issue description makes it look like I'm waiting on it.  I wasn't planning on waiting on the messaging stuff, either but implement that later.

Right now I'm just waiting on an SVG for that down-caret.  The one I downloaded from Zeplin appears blank.  Would you mind verifying that, @micahalcorn?

**Comment by micahalcorn** [comment]
Hmm, yeah, something weird is going on with it. Maybe try to rotate and use https://github.com/OriginProtocol/origin/blob/master/dapps/marketplace/public/images/caret-dark.svg ♻️

**Comment by mikeshultz** [comment]
ok, I used your suggestion.  Looks okay to me!  PR #2334 is ready for review.

Will leave this issue open to track and work on the messaging change as I have time.

**Comment by micahalcorn** [comment]
Moved the the last task to a separate issue 👍
